Source/scrollrt.cpp

```diff
@@ -148,20 +155,30 @@ static void scrollrt_draw_cursor_item()
 	assert(!sgdwCursWdt);
 
 	if (pcurs <= CURSOR_NONE || cursW == 0 || cursH == 0) {
+		if (hwcursor)
+			SDL_ShowCursor(SDL_DISABLE);
 		return;
 	}
 
 	if (sgbControllerActive && !IsMovingMouseCursorWithController() && pcurs != CURSOR_TELEPORT && !invflag && (!chrflag || plr[myplr]._pStatPts <= 0)) {
+		if (hwcursor)
+			SDL_ShowCursor(SDL_DISABLE);
 		return;
 	}
 
 	mx = MouseX - 1;
+	if (hwcursor) {
+		mx = 0;

```

Comment:
The cursor position is set to a constant value to ensure we always blit the cursor into the same position in the framebuffer so that we can read it from there and there's no jitter involved.

Reply:
Is there any way to better represent that explanation in the code itself?